src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml

```diff
@@ -11,662 +11,340 @@
-          [1685009176.921418][39681:39683] CHIP:DMG: },
-          [1685009176.921421][39681:39683] CHIP:DMG:
-          [1685009176.921423][39681:39683] CHIP:DMG: },
-          [1685009176.921428][39681:39683] CHIP:DMG:
-          [1685009176.921430][39681:39683] CHIP:DMG: AttributeStatusIB =
-          [1685009176.921432][39681:39683] CHIP:DMG: {
-          [1685009176.921434][39681:39683] CHIP:DMG: AttributePathIB =
-          [1685009176.921436][39681:39683] CHIP:DMG: {
-          [1685009176.921438][39681:39683] CHIP:DMG: Endpoint = 0x0,
-          [1685009176.921440][39681:39683] CHIP:DMG: Cluster = 0x3f,
-          [1685009176.921443][39681:39683] CHIP:DMG: Attribute = 0x0000_0000,
-          [1685009176.921445][39681:39683] CHIP:DMG: ListIndex = Null,
-          [1685009176.921447][39681:39683] CHIP:DMG: }
-          [1685009176.921450][39681:39683] CHIP:DMG:
-          [1685009176.921452][39681:39683] CHIP:DMG: StatusIB =
-          [1685009176.921456][39681:39683] CHIP:DMG: {
-          [1685009176.921458][39681:39683] CHIP:DMG: status = 0x00 (SUCCESS),
-          [1685009176.921462][39681:39683] CHIP:DMG: },
-          [1685009176.921464][39681:39683] CHIP:DMG:
-          [1685009176.921466][39681:39683] CHIP:DMG: },
-          [1685009176.921470][39681:39683] CHIP:DMG:
-          [1685009176.921472][39681:39683] CHIP:DMG: AttributeStatusIB =
-          [1685009176.921475][39681:39683] CHIP:DMG: {
-          [1685009176.921477][39681:39683] CHIP:DMG: AttributePathIB =
-          [1685009176.921479][39681:39683] CHIP:DMG: {
-          [1685009176.921481][39681:39683] CHIP:DMG: Endpoint = 0x0,
-          [1685009176.921483][39681:39683] CHIP:DMG: Cluster = 0x3f,
-          [1685009176.921485][39681:39683] CHIP:DMG: Attribute = 0x0000_0000,
-          [1685009176.921488][39681:39683] CHIP:DMG: ListIndex = Null,
-          [1685009176.921491][39681:39683] CHIP:DMG: }
-          [1685009176.921494][39681:39683] CHIP:DMG:
-          [1685009176.921496][39681:39683] CHIP:DMG: StatusIB =
-          [1685009176.921499][39681:39683] CHIP:DMG: {
-          [1685009176.921501][39681:39683] CHIP:DMG: status = 0x00 (SUCCESS),
-          [1685009176.921503][39681:39683] CHIP:DMG: },
-          [1685009176.921506][39681:39683] CHIP:DMG:
-          [1685009176.921508][39681:39683] CHIP:DMG: },
-          [1685009176.921511][39681:39683] CHIP:DMG:
-          [1685009176.921513][39681:39683] CHIP:DMG: ],
-          [1685009176.921519][39681:39683] CHIP:DMG:
-          [1685009176.921521][39681:39683] CHIP:DMG: InteractionModelRevision = 1
-          [1685009176.921523][39681:39683] CHIP:DMG: }
-          [1685009176.921551][39681:39683] CHIP:DMG: WriteClient moving to [AwaitingDe]
-      disabled: true
+      cluster: "Group Key Management"
+      command: "writeAttribute"
+      attribute: "GroupKeyMap"
+      arguments:
+          value:
+              [
+                  {
+                      FabricIndex: FabricIndexValue,
```

Comment:
Just use 0 for FabricIndex, if YAML is forcing you to provide it.  This value is not sent to the server anyway, per spec.... (and you will note the test plan does not mention it).

Reply:
Updated.